cmd/opm/alpha/bundle/build.go

```diff
@@ -76,12 +78,14 @@ func newBundleBuildCmd() *cobra.Command {
 	bundleBuildCmd.Flags().StringVarP(&outputDirArgs, "output-dir", "u", "",
 		"Optional output directory for operator manifests")
 
+	bundleBuildCmd.Flags().BoolVarP(&skipValidationArgs, "skip-validation", "v", false,
+		"Skips validating ClusterServiceVersion present in the (output) bundle directory")
```

Comment:
yes. The validation for everything else is under `validation` cmd. But i wonder if we want auto csv validation here because CSV is generated based on the bundle and things can miss.

Reply:
Hmm, If validation is for everything, I don't think this should call out the CSV specifically in the subscription; i.e. it doesn't seem like there's much difference between `validate(CSV) + validate(everything)` and `don't_validate(CSV) + validate(everything)`.

Reply:
The action related to this command does only validate CSV at the moment. I wonder if that is what we want. It almost does not make any sense to just validate CSV here. We would want the bundle to be valid. Because the CSV is generated, I put in the validation here as suggested. But in reality, I think we can separate the cmd logic. 

The other option is to add crd+buundle+CSV validation here so that if we generate CSV, we validate the entire bundle. 

The validation cmd mentioned in the previous comment is `opm alpha bundle validate`.